doc/classes/PhysicsServer3D.xml

```diff
@@ -44,7 +44,8 @@
 		<method name="area_create">
 			<return type="RID" />
 			<description>
-				Creates an [Area3D].
+				Creates an [Area3D] object in the physics server, and returns the [RID] that identifies it. The default settings for the created area include a collision layer and mask set to [code]1[/code], and [code]monitorable[/code] set to [code]false[/code].
```

Comment:
```suggestion
				Creates a 3D area object in the physics server, and returns the [RID] that identifies it. The default settings for the created area include a collision layer and mask set to [code]1[/code], and [code]monitorable[/code] set to [code]false[/code].
```
To make it a bit more clear that this area *isn't* an `Area3D` and match the 2D text better